content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -79,7 +79,8 @@ For a reference to old feature gates that are removed, please refer to
 | `CSIMigrationPortworx` | `false` | Alpha | 1.23 | 1.24 |
 | `CSIMigrationPortworx` | `false` | Beta | 1.25 | |
 | `CSINodeExpandSecret` | `false` | Alpha | 1.25 | 1.26 |
-| `CSINodeExpandSecret` | `true` | Beta | 1.27 | |
+| `CSINodeExpandSecret` | `true` | Beta | 1.27 | 1.28 |
+| `CSINodeExpandSecret` | `true` | GA   | 1.29 | |
```

Comment:
GA'ed feature should be moved to the second table.

Reply:
Sure.. thanks! @tengqm